apps/terminal/serializers/terminal.py

```diff
@@ -147,3 +147,8 @@ class ConnectMethodSerializer(serializers.Serializer):
     type = serializers.CharField(max_length=128)
     endpoint_protocol = serializers.CharField(max_length=128)
     component = serializers.CharField(max_length=128)
+
+
+class EncryptedConfigSerializer(serializers.Serializer):
+    secret_encrypt_key = serializers.CharField(max_length=128)
+    encrypted_value = serializers.CharField(max_length=128)
```

Comment:
There are no major issues detected. The ConnectMethodSerializer has some minor differences when compared to its previous version from 2021-09-01:

  * The first line (`class ConnectMethodSerializer(serializers.Serializer):`) is not changed.
  * The class name `serializers.Serializer` remains unchanged throughout the file.

For the EncryptedConfigSerializer, the main point of difference is that it now includes two new fields: `secret_encrypt_key`, which takes a string with max length 128 characters, and `encrypted_value`, with a similar maximum size (128). This suggests more specific encryption-related data may be added here.